pkg/uefivars/boot/efiDppHardware.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+package boot
+
+import (
+	"fmt"
+)
+
+type EfiDppHwSubType EfiDevPathProtoSubType
```

Comment:
oof, after the Uuid comment I realize that a lot of these are not Go-idiomatic, but they would turn into kind of unreadable initialisms if you capitalized everything according to https://github.com/golang/go/wiki/CodeReviewComments#initialisms

I'd say we can leave them all as is against the style recommendation; it'd be more confusing if this was `EFIDPPHWSubType` imho. Just documenting that I thought about this here.